### ISSUE TITLE
CellLocate AuthenticationToken not set

### ISSUE THREAD

**Issue by eeFLis** [u-blox/ubxlib#129]
Hi there

We use `uLocationGetStart()` with location type = U_LOCATION_TYPE_CLOUD_CELL_LOCATE.
The problem we see is that the auth token is never set.

in `setUgsrv()` the auth token is always read first.
if none was set before the auth token ends in "not set".

should not:

`uAtClientWriteString(atHandle, authenticationTokenStr, true);`

replaced by

```
if (pAuthenticationTokenStr != NULL) {
    uAtClientWriteString(atHandle, pAuthenticationTokenStr, true);
} else {
    uAtClientWriteString(atHandle, authenticationTokenStr, true);
}
```
I have created a pull request for this change

regards

**Comment by RobMeades** [comment]
Hi @eeFLis and sorry about this: looking into the code, it seems I lost the setting of the Cell Locate authentication token when I introduced the configuration of AssistNow support a few months ago and, at the same time, I added WiFi Location support during which I unintentionally removed the Cell Locate authentication tokens we use during testing from the test system, so Cell Locate wasn't being tested.  Ugh.

Let me push the change to restore the Cell Locate testing out to `master` here, pull your PR into our test system and give it a run-through, then if no issues I can merge it across.
